### ISSUE TITLE
AUR package does not build

### ISSUE THREAD

**Issue by jrabensc** [eplatonoff/pilorama#16]
Hi, I'm not sure if this is the right place to report but the AUR package is currently not building.

Here's the log:

```
Preparing...
Cloning pilorama-git build files...
Checking pilorama-git dependencies...
Resolving dependencies...
Checking inter-conflicts...

Building pilorama-git...
==> Making package: pilorama-git v3.0.1.r1.g1843d7b-1 (Mo 07 Sep 2020 12:37:54 CEST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating pilorama-git git repo...
Fetching origin
==> Validating source files with sha256sums...
    pilorama-git ... Skipped
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Creating working copy of pilorama-git git repo...
Cloning into 'pilorama-git'...
done.
==> Starting pkgver()...
==> Removing existing $pkgdir/ directory...
==> Starting build()...
Cannot find file: src/pilorama.pro.
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by hunterbdean** [comment]
I have no idea how github etiquette works so I'm sorry if this isn't how it's done, but this issue seems to be due to an errant capital P in:

`pilorama/src/Pilorama.pro`

replace the capital P in that filename with a lowercase one and get back to being productive :)

**Comment by sergeykuzmich** [comment]
Fixed in #17
